docs/mobile/state-management.md

```diff
@@ -0,0 +1,7 @@
+## Mobile Playbook: State management
+
+#### Persist Redux/Flux store to disk to survive shutdowns and tombstoning
+
+#### Prepare for store schema migrations/pruning on updates
+
+#### Using GraphQL with Redux vs Relay
```

Comment:
I don't have enough experience to speak to MobX. I'd love a paragraph on it if you have time to write one. Feel free to drop a commit into https://github.com/FormidableLabs/formidable-playbook/pull/48